packages/components/src/local/organisms/planning-messages-list/index.tsx

```diff
@@ -218,6 +218,8 @@ export const PlanningMessagesList: React.FC<PropTypes> = ({
         return msg
       })
       console.log('Archiving:', markArchived)
+      // if (markArchived.length === 0) return null
```

Comment:
@lilitkarapetyan - this isn't working, but it's in my code, not yours :-(

At this point in the code, the `pendingArchive` array is empty.  I've inserted debug lines to capture the array getting set (in `archiveSelected`), but in `archiveConfirmed` the array is empty.  React State...

I'd appreciate you helping out with this issue, please.

Reply:
I tested there with `rows` data,